<?php

namespace ContainerXN1QCcy;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_Hi_GBQzService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.hi.gBQz' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.hi.gBQz'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'doctrine' => ['services', 'doctrine', 'getDoctrineService', false],
            'jsonesponse' => ['privates', '.errored..service_locator.hi.gBQz.Symfony\\Component\\HttpFoundation\\JsonResponse', NULL, 'Cannot autowire service ".service_locator.hi.gBQz": it references class "Symfony\\Component\\HttpFoundation\\JsonResponse" but no such service exists.'],
            'validator' => ['services', '.container.private.validator', 'get_Container_Private_ValidatorService', false],
        ], [
            'doctrine' => '?',
            'jsonesponse' => 'Symfony\\Component\\HttpFoundation\\JsonResponse',
            'validator' => '?',
        ]);
    }
}
